src/pages/iou/steps/IOUParticipantsPage/IOUParticipantsPage.js

```diff
@@ -63,12 +70,14 @@ const IOUParticipantsPage = (props) => {
                 onStepComplete={props.onStepComplete}
                 participants={props.participants}
                 onAddParticipants={props.onAddParticipants}
+                safeAreaPaddingBottomStyle={props.safeAreaPaddingBottomStyle}
             />
         )
         : (
             <IOUParticipantsRequest
                 onStepComplete={props.onStepComplete}
                 onAddParticipants={props.onAddParticipants}
+                safeAreaPaddingBottomStyle={props.safeAreaPaddingBottomStyle}
             />
```

Comment:
Why are we passing this?

Reply:
It's never used there

Reply:
Do you prefer wrapping `SafeAreaConsumer` in child view or passing `safeAreaPaddingBottomStyle` prop from parent?
IOUParticipantsSplit used SafeAreaConsumer but not on IOUParticipantsRequest.
Based on preference, I will update code accordingly to make them consistent.

Reply:
Less DRY is preferred!

Reply:
Both have drawbacks.
wrapping `SafeAreaConsumer` in child costs another insets calculation from SafeAreaInsetsContext which is redundant.
passing prop to child multiple times is not DRY code but performant than wrapping `SafeAreaConsumer` multiple times.